src/DynamoCoreWpf/Views/PackageManager/Controls/PackageManagerPublishControl.xaml.cs

```diff
@@ -32,17 +32,43 @@ public PackageManagerPublishControl()
             InitializeComponent();
 
             this.Loaded += InitializeContext;
+            this.DataContextChanged += ThisDataContextChanged;
```

Comment:
Following the pattern in our code base, can we change the event name to OnDataContextChanged or PackageManagerPublishControl_DataContextChanged